crates/fluvio-cluster/src/cli/delete.rs

```diff
@@ -16,13 +20,43 @@ pub struct DeleteOpt {
     #[arg(long, conflicts_with = "local", conflicts_with = "sys")]
     k8: bool,
 
+    /// Delete system chart
     #[arg(long, conflicts_with = "k8", conflicts_with = "local")]
-    /// delete system chart
     sys: bool,
+
+    /// Do not prompt for confirmation
+    #[arg(long)]
+    force: bool,
 }
 
 impl DeleteOpt {
     pub async fn process(self) -> Result<(), ClusterCliError> {
+        if !self.force {
+            let config_file = ConfigFile::load_default_or_new()?; // NOTE: Not sure we want `or_new` here
+            let (current_cluster, current_profile) = {
+                let config = config_file.config();
+                (config.current_cluster()?, config.current_profile()?)
+            };
+
+            // Prompt for confirmation
+            println!(
+                "WARNING: You are about to delete {cluster}/{endpoint}. This operation is irreversible \
+                and the data stored in your cluster will be permanently lost. \
+                \nPlease type the cluster name to confirm: {cluster} <enter> (to confirm) / or CTRL-C (to cancel)",
```

Comment:
My bad, I should have made this clear:

`Please type the cluster name to confirm:`

should be a prompt where the user needs to type:
 > <ClusterName> <enter> 
or 
> CTRL-C to cancel